awx/main/management/commands/replay_job_events.py

```diff
@@ -76,9 +91,10 @@ def get_job_events(self, job):
             job_events = job.inventory_update_events.order_by('created')
         elif type(job) is SystemJob:
             job_events = job.system_job_events.order_by('created')
-        if job_events.count() == 0:
+        count = job_events.count()
+        if count == 0:
             raise RuntimeError("No events for job id {}".format(job.id))
-        return job_events
+        return job_events, count
 
```

Comment:
this makes sense, a few places in the command unit test might need to be updated to anticipate the additional return vals